app/forms/hyrax/video_form.rb

```diff
@@ -1,9 +1,10 @@
 # Generated via
 #  `rails generate hyrax:work Video`
 module Hyrax
-  class VideoForm < Hyrax::Forms::WorkForm
+  class VideoForm < Tufts::HasTranscriptForm
```

Comment:
So I realize it's DRYer to structure this in this way and keep the transcript stuff together, but I feel like in the long run it might be clearer for all the Tufts works to inherit from `Hyrax::Forms::WorkForm` than having some that do and some that don't.  The way it's done right here obscures the fact that Audio & Video works are more similar to other work types than different: i.e. they behave exactly like all other WorkForms plus some behavior.  Would it be possible to include the additional form behaviors as mixed-in modules instead?